Simplify the usage of the "message" term (#39)
ietf-wg-asdf/instance-information: draft-ietf-asdf-instance-information.md

Author: Maybe we could just speak of “messages” or “types of messages” instead of “message archetypes” here and throughout the document?

# Instance Information and SDF

The instantiation of an SDF model does not directly express an instance, which is, for example, a physical device or a digital twin.

> Instead, the instantiation produces an instance-related *message*, which adheres to a common message format and is always controlled by the corresponding SDF model.

Author: I think we might be able to further simplify this sentence:

```suggestion
Instead, the instantiation produces an instance-related *message*, whose format is always controlled by the corresponding SDF model.
```

    </tr>
  </tbody>
</table>
{: #instance-message-dimensions title="Systematization of messages along the dimensions \"Content\" and \"(Intended) Effect\"."}

> The common message format can be used for all four message archetypes.

Author: ... taking the previous comment into account, we then simply remove this sentence, I think.
